packages/create-webiny-project/index.js

```diff
@@ -15,15 +15,17 @@ console.log = str => originalLog(indent(str, 2));
 
 const packageJson = require("./package.json");
 const init = require("./init.js");
+const rimraf = require("rimraf");
 
 yargs
     .usage("Usage: $0 <project-name> [options]")
     .version(packageJson.version)
     .demandCommand(1)
-    .example("$0 my-project --template=full")
+    .example("$0 my-project")
     .example(
         "$0 create-webiny-project --template=../path/to/template --tag=../path/to/webiny/files"
     )
+    .example("$0 create-webiny-project --log=./my-logs.txt")
```

Comment:
@EmilK15 just caught my eye that these command examples look weird. Looks like they're missing the project name parameter. If we consider `$0` to be `create-webiny-project`, then having a project name `create-webiny-project` looks weird, and in the examples above the `create-webiny-project` string is not even mentioned, so it refers to `$0`.